src/dns-test.js

```diff
@@ -123,6 +134,62 @@ function encodeTCPQuery(domain, rrtype, dnssec_ok) {
     return buf;
 }
 
+/**
+ * Generate a random sub-domain under a domain name we control to query using 
+ * dns.resolve()
+ */
+function createRandomDomain(domain) {
+    // Generate random values
+    let typedArray = new Uint8Array(16);
+    let randomValues = crypto.getRandomValues(typedArray);
```

Comment:
Fyi getRandomValues modifies the input typed array and returns the same array. No need for a separate variable name.